cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -1030,8 +1039,12 @@ instance ToObject (TraceTxSubmissionInbound txid tx) where
 -- TODO: ouroboros-network should provide a newtype wrapper for 'SockAddr'.
 instance ToObject Socket.SockAddr where
     toObject _verb sockAddr =
-      mkObject [ "sockAddr" .= String (pack $ show sockAddr) ]
+      mkObject [ "sockAddr" .= toJSON sockAddr ]
```

Comment:
I'd prefer if we separated the address from the port. See ac7cf197f .

Reply:
ok, I will also need to change the encoding of  `Map SockAddr AbstractState` to a list of triples `(addr, port, state)`.

Reply:
Done